src/main/java/org/anstreth/schedulebot/schedulerformatter/SimpleLessonFormatter.java

```diff
@@ -2,23 +2,34 @@
 
 import org.anstreth.ruzapi.response.Auditory;
 import org.anstreth.ruzapi.response.Lesson;
+import org.anstreth.ruzapi.response.Teacher;
 import org.springframework.stereotype.Component;
 
 @Component
-class SimpleLessonFormatter implements LessonFormatter{
+class SimpleLessonFormatter implements LessonFormatter {
 
     @Override
     public String formatLesson(Lesson lesson) {
-        return String.format("%s - %s: %s (%s)\n%s",
+        return String.format("%s - %s: %s (%s)\n[%s]\n%s",
```

Comment:
I guess it would be better to not show braces at all when there's no teacher for this class

Reply:
Yep, agree. I'll do the same with auditories then